packages/ui/src/ButtonGroup/index.tsx

```diff
@@ -88,24 +96,31 @@ export const ButtonGroup = ({
       */}
       {isIconOnly(props) &&
         props.buttons.map((button, index) => (
-          <ButtonPriorityContext.Provider key={index} value={index === 0 ? 'primary' : 'secondary'}>
```

Comment:
We got rid of the context, so now we just pass the priority directly to the button.